assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/clip/clip_mlflow_wrapper.py

```diff
@@ -121,11 +126,8 @@ def predict(self, context: mlflow.pyfunc.PythonModelContext, input_data: pd.Data
 
     def run_inference_batch(
```

Comment:
The call to this function also needs to be updated right ? Can you run the model publish pipeline for CLIP zero shot with this changes and check if it works.

Reply:
updated and linked clip model publish pipeline for zero shot in description